src/lib/Libtpp/tpp_common.h

```diff
@@ -273,6 +273,7 @@ enum TPP_MSG_TYPES {
 #define TPP_STRM_TIMEOUT        600
 #define TPP_MIN_WAIT            2
 #define TPP_SEND_SIZE           8192
+#define TPP_COMPR_SIZE			8192
```

Comment:
is the value of TPP_COMPR_SIZE properly indented?
TPP_SEND_SIZE and TPP_COMPR_SIZE - have the same value.
How is using TPP_COMPR_SIZE instead of TPP_SEND_SIZE helping?

Reply:
fixed the indentation.

TPP_SEND_SIZE was being overloaded. Used for deciding the packet size, beyond which we will break down packets into multiple. Using the same value for the compression threshold meant we could not set it to a smaller value than the packet size. For example, we might want to send 16k long packets each, but compress if the packet size > 8k. Two separate variables allow this - and makes for easy testing